### ISSUE TITLE
[CAL-3966] Pipedrive interation does not link

### ISSUE THREAD

**Issue by joseuo94** [calcom/cal.com#15531]
When installing integration the app does not open the new tab to log into the Pipedrive account.

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-3966](https://linear.app/calcom/issue/CAL-3966/pipedrive-interation-does-not-link)</sub>

**Comment by Neon-20** [comment]
https://github.com/calcom/cal.com/assets/55043383/a9e8cb48-0034-4ae1-a0a6-be36b684d33d

Referring to this error? 
@joseuo94

**Comment by joseuo94** [comment]
@Neon-20 Yes. it is supposed to open a pop-up with a different window where customers can log in with their pipedrive account.

**Comment by JakobStadlhuber** [comment]
I have the same problem

**Comment by SeveHo** [comment]
+1
